src/nvim/ex_docmd.c

```diff
@@ -6861,7 +6905,26 @@ void ex_cd(exarg_T *eap)
     if (new_dir == NULL || vim_chdir(new_dir))
       EMSG(_(e_failed));
     else {
-      post_chdir(eap->cmdidx == CMD_lcd || eap->cmdidx == CMD_lchdir);
+      int scope = 0; // Depends on what command was invoked
```

Comment:
I think that initialising the scope to 0 was a mistake though, I should set it to something invalid, like -1.